list.go

```diff
@@ -271,6 +276,18 @@ NextFile:
 							p.IgnoredGoFiles = append(p.IgnoredGoFiles, fname)
 							continue NextFile
 						}
+						matchesPositive, err := regexp.MatchString("go\\d+\\.\\d+", b)
+						if err != nil {
+							return err
+						}
+						if matchesPositive && b != version {
+							p.IgnoredGoFiles = append(p.IgnoredGoFiles, fname)
+							continue NextFile
+						}
+						if b == ("!" + version) {
```

Comment:
I *think* negatives also work the same way. eg. a tag of `!go1.5` means NOT go1.5 OR go1.6 or go1.7, etc.